app/workers/github/create_issue_job.rb

```diff
@@ -4,21 +4,15 @@
 module Github
   class CreateIssueJob
     include Sidekiq::Worker
-
-    # limit the rate at which Github API calls are made
-    # so as not to trigger Github spam protections
-    sidekiq_options queue: 'low',
-                    rate: {
-                      name: 'prevent_feedback_spam',
-                      limit: 20,
-                      period: 3600, ## An hour
-                    }
+    THROTTLE = Sidekiq::Limiter.window('prevent_feedback_spam', 4, :minute).freeze
```

Comment:
I think 20/hr 4/min was for Github and to stop the form being spammed? Now that we have Rack level rate limiting can this be set to the Github authenticated request limit?

Reply:
I'm not sure what the answer is.  Our call from `vets-api` to GH API, is _authenticated_ so abiding by their rate limits makes sense.  OTOH, our call to `GET /v0/feedback` is not authenticated so a part of me wants to be closer to 60/hour on the `vets-api`--->GH API side.

What I _do_ know, is that do not want GH to block us with spam protections, which in my testing happened after ~65 identical `create_issue` calls back to back.  The testing was under extreme conditions but I think it can be used as a baseline.

Reply:
I think the answer is somewhere in the middle.  @jkassemi noted that at our peak, we had 24 feedback submissions in one hour.  @ 4/minute = 240/hour which seemed like a reasonably high number.  It also accomplished the goal of _gradually_ sending `create_issue` requests to GH api as opposed to slamming them with a high number of requests all at once.

Reply:
Perhaps we try some number / rate limit and add StatsD monitoring that tracks our [`Octokit::Client.rate_limit.remaining`](https://octokit.github.io/octokit.rb/Octokit/Client/RateLimit.html).  We can monitor and adjust accordingly.

Reply:
While we'd be okay with GH by increasing this to the authenticated request limit, it might be harder to clean up our issue tracker if someone works around our vets-api rate limit (this is as simple as hitting us from multiple IPs).

I like the idea of monitoring the rate limit (a `GitHubStatsJob` to export that value). We should also have a playbook for dealing with a situation where we're swamped with feedback. Can we easily disable this feature if we need to while we clean up and assess?